authHelpers/authRoutes.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const session = require('express-session');
+
+
+const db = require('../data/dbConfig.js');
+
+const router = express.Router();
+
+const sessionConfig = {
+  secret: 'nobody-tosses.a%dwarf.!',
+  name: 'user-session',
+  httpOnly: true,
+  resave: false,
+  saveUninitialized: false,
+  cookie: {
+    secure: false,
+    maxAge: 1000 * 60 * 1
+  },
+};
+
+router.use(session(sessionConfig));
+
+// register a new user
+router.post('/register', isAuthenticated, (req, res) => {
```

Comment:
When registering a user, they wouldn't be authenticated, so why have `isAuthenticated` middleware be used here?